wolox-training/igc-rails
# Devise

### Summary
- Added Devise for user sign up.
- Changed HTTP basic auth for Devise auth

### Trello card
https://trello.com/c/0lOGxZUf/9-instalar-y-configurar-devise

### Test plan
1. Click on sign up
2. Fill in the details and accept
3. User should now be signed in.
4. Sign out
5. Sign in as the previously created user (if wrong credentials are submitted, it should fail) 
6. Attempting to create and destroy posts and comments should work.
7. Sign out
8. Attempting to create and destroy posts and comments shouldn't work, redirecting to the sign in page instead.
